learn/core_concepts/primary_key.md

```diff
@@ -232,3 +232,31 @@ This happens when your document id does not have the correct [format](#formattin
     "finishedAt": "2021-12-30T11:28:59.084803Z"
 }
 ```
+
+### `index_primary_key_multiple_candidates_found`
+
+This error occurs when Meilisearch fails to guess your primary key as the index has multiple attributes containing `id`. It can be resolved by [manually setting the index's primary key](#setting-the-primary-key-on-document-addition).
```

Comment:
Do you see why I am suggesting these changes?

```suggestion
This error occurs when you add documents to an index for the first time and Meilisearch finds multiple attributes ending with `id`. It can be resolved by [manually setting the index's primary key](#setting-the-primary-key-on-document-addition).
```

Reply:
The sentence paraphrases the error and isn't useful

Reply:
The main thing is that the difference between "containing id" and "ending with id" is potentially huge; it's the difference between having an accurate understanding and expectation of the inference, and not.

Reply:
The other thing is that saying "the index has multiple attributes containing `id`" is incorrect and potentially confusing. When the error occurs, the document addition fails, so the index is empty.